api/src/test/resources/io/strimzi/api/kafka/model/KafkaConnector-with-filestream-sink.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: KafkaConnector
+metadata:
+  name: local-file-sink
+  labels:
+    connect-cluster: my-connect-cluster
+spec:
+  class: org.apache.kafka.connect.file.FileStreamSinkConnector
+  tasksMax: 1
+  config:
+    file: "/tmp/test.sink.txt"
+    topics: "connect-test"
```

Comment:
I see "topics" here (plural) and "topic" in the other examples. Why?

Reply:
Mistake.

Reply:
I don't think it matters that much because the `config` is just a map. But AFAIK it is `topic` for source connectors and `topics` or `topics.regex` for sink connectors.